packages/synapse-react-client/src/components/OAuthClientManagement/CreateOAuthClient.integration.test.tsx

```diff
@@ -47,18 +56,68 @@ function renderComponent(props: CreateOAuthModalProps = defaultProps) {
 
 function setUp(props: CreateOAuthModalProps = defaultProps) {
   const user = userEvent.setup()
+
+  // useFakeTimers to ensure that isOAuthClientReverificationRequired is called
+  // at least once per test, since the call is wrapped in useDebouncedEffect
+  jest.useFakeTimers()
```

Comment:
Use fake timers, so that `isOAuthClientReverificationRequired` is called at least once per test, then switch back to using real timers for the rest of the test. This seems to resolve the flakiness in local test runs.